api/GovStack_Registration_BB_API.json

```diff
@@ -153,17 +149,17 @@
 			}
 		},
 
-		"/service/{serviceKey}/startWithForm": {
+		"/services/{serviceKey}/applications": {
```

Comment:
if we are following the REST standards (which are widely used in the industry), URLs should reflect "resources" not actions and there can be "sub-resources". Here e.g. applications belonging to a specific service